CHANGES.txt

```diff
@@ -106,6 +106,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       not be cached because the emitted filename contained a '$' and when
       looked up through a node ended up generating a Python SyntaxError
       because it was passed through scons_subst().
+    - Have the gfortran tool do a better job of honoring user preferences
```

Comment:
Can you add the whole list here? 
['F77', 'F90', 'F95', 'F03', 'F08']
And SHF77,.. as well?

I like to have CHANGES.txt such that a user could search for the variable/tool/etc  they're using when they hit a problem to either see when it broke or when it was fixed.

Also a blurb in RELEASE.txt since this will change behavior.  And remove doesn't change behavior, but rather it changes behavior to be in line with how tools are expected to work..?

Reply:
The "behavior does not change" line was in respect to the tests.  I'll fiddle the blurb and add something to RELEASE.